server/models/services/AtService.js

```diff
@@ -184,6 +184,28 @@ const removeAt = async (id, deleteOptions = { truncate: false }) => {
 
 // AtVersion
 
+/**
+ * You can pass any of the attribute arrays as '[]' to exclude that related association
+ * @param {number} id - unique id of the AtVersion model being queried
+ * @param {string[]} atAttributes  - At attributes to be returned in the result
+ * @param {object} options - Generic options for Sequelize
+ * @param {*} options.transaction - Sequelize transaction
+ * @returns {Promise<*>}
+ */
+const getAtVersionById = async (
+    id,
+    atAttributes = AT_ATTRIBUTES,
+    options = {}
+) => {
+    return ModelService.getById(
+        AtVersion,
+        id,
+        atAttributes,
```

Comment:
I think line 203 should be atVersionAttributes not atAttributes.

Reply:
Good catch!

Reply:
Good catch!